src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const people = document.querySelectorAll('li');
+
+function sortList(arr) {
+  function transNum(num) {
+    return Number(num.replace('$', '').split(',').join(''));
+  }
+
+  return arr.map(item => {
+    const dataInfo = item.dataset;
+    const innerText = `${item.innerText}`;
+
+    return {
+      ...dataInfo,
+      name : innerText
+    }
+  }).sort((a, b) => transNum(b.salary) - transNum(a.salary));
+}
+const sortValue = sortList([...people]);
```

Comment:
please use better naming for variables
```suggestion
const sortedListValues = sortList([...people]);
```